modules/highlight-mfa-users/class-highlight-mfa-users.php

```diff
@@ -89,6 +89,37 @@ public static function register_hooks() {
 		add_action( 'set_user_role', [ __CLASS__, 'clear_mfa_count_cache_for_user_role_change' ], 10, 1 );
 		add_action( 'add_user_role', [ __CLASS__, 'clear_mfa_count_cache_for_user_role_change' ], 10, 1 );
 		add_action( 'remove_user_role', [ __CLASS__, 'clear_mfa_count_cache_for_user_role_change' ], 10, 1 );
+
+		// Add SDS hook
+		add_filter( 'vip_site_details_index_security_boost_data', [ __CLASS__, 'add_users_without_2fa_count_to_sds_payload' ] );
+	}
+
+	/**
+	 * Add the users_without_2fa_count to the SDS payload.
+	 *
+	 * @param array $data The SDS payload data.
+	 * @return array The modified SDS payload data.
+	 */
+	public static function add_users_without_2fa_count_to_sds_payload( $data ) {
+		// Add fix for unreliable FOUND_ROWS() query
+		add_filter( 'found_users_query', [ Users_Query_Utils::class, 'fix_found_users_query' ], 10, 2 );
+
+		$users_without_2fa_count = self::get_mfa_disabled_count();
+
+		$data['users_without_2fa_count'] = $users_without_2fa_count;
+
+		if ( is_multisite() ) {
+			// Get number of users without 2FA for all blogs (network-wide with blog_id = 0)
+			$users_without_2fa_count_all_blogs = self::get_mfa_disabled_count( 0 );
+
+			// Add network-wide users without 2FA count to the SDS payload
+			$data['users_without_2fa_count_all_blogs'] = $users_without_2fa_count_all_blogs;
+		}
+
+		// Remove fix for unreliable FOUND_ROWS() query
+		remove_filter( 'found_users_query', [ Users_Query_Utils::class, 'fix_found_users_query' ], 10 );
```

Comment:
I think we should pass the priority here to remove the filter completely.

```suggestion
		remove_filter( 'found_users_query', [ Users_Query_Utils::class, 'fix_found_users_query' ], 10, 2 );
```

Reply:
I'm already passing the priority. What I'm not passing is the argument count, which [remove_filter](https://developer.wordpress.org/reference/functions/remove_filter/) doesn't accept as an argument.

Reply:
You're absolutely right, I miscounted. 😬